DmfSamples/InterfaceSample1/InterfaceSample1.vcxproj

```diff
@@ -31,31 +31,15 @@
     <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
-  <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+  <PropertyGroup Label="Configuration" Condition="'$(Configuration)'=='Release'">

```

Comment:
Most of these settings are applicable for debug also.

Reply:
Yes, i left put a Debug section at line 42